data/free worlds start.txt

```diff
@@ -1615,6 +1774,122 @@ mission "FW Pirates 4.1"
 
 
 
+mission "FW Pirates 4.2"
+	landing
+	name "Senate Hearing"
+	description "Report to <destination>, for a Senate hearing on your recent failure to obey their orders. If you do not arrive by <date>, your service will be terminated."
+	source "Longjump"
+	destination "Bourne"
+	deadline 14
+	to offer
+		has "fw pirates: crushed"
+	on fail
+		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	
+	on offer
+		conversation
+			`JJ and Freya meet you on your ship soon after you land on <origin>, accompanied by two uniformed military policemen. 
+			`	"We heard about the hornet's nest you stirred up from the other captains, before we arrested them," says JJ. "Congratulations. you escaped from the disaster brought on by your stupidity, and the pirates are united against us. Now we have to fight a war on two fronts while you sit in prison with Tomek." He gestures to two military policemen, who pull out handcuffs and move toward you.`
+
+			choice
```

Comment:
I've revised the choices to make them shorter and more indicative of a matter-of-fact tone, a passive submission, and an in-your-face tone.